addons/account/static/src/views/account_dashboard_kanban/account_dashboard_kanban_renderer.js

```diff
@@ -0,0 +1,44 @@
+import { KanbanRenderer } from "@web/views/kanban/kanban_renderer";
+import { DashboardKanbanRecord } from "./account_dashboard_kanban_record";
+
+import { useSubEnv, reactive } from "@odoo/owl";
+
+export class DashboardKanbanRenderer extends KanbanRenderer {
+    static template = "account.DashboardKanbanRenderer";
+    static components = {
+        ...KanbanRenderer.components,
+        KanbanRecord: DashboardKanbanRecord,
+    };
+
+    setup() {
+        super.setup();
+        useSubEnv({
+            dashboardState: reactive({isDragging: false}),
+            disableDragging: this.disableDragging.bind(this),
+        });
+    }
+
+    kanbanDragEnter(e) {
+        this.env.dashboardState.isDragging = true;
+    }
+
+    kanbanDragLeave(e) {
+        const mouseX = e.clientX, mouseY = e.clientY;
+        const {x, y, width, height} = this.rootRef.el.getBoundingClientRect();
+        if (!(mouseX > x && mouseX <= x + width && mouseY > y && mouseY <= y + height)) {
+            // if the mouse position is outside the kanban renderer, all cards should hide their dropzones.
+            this.disableDragging();
+        } else {
+            this.env.dashboardState.isDragging = true;
+        }
+    }
+
+    kanbanDragDrop(e) {
+        this.disableDragging();
+        return false;
+    }
+
+    disableDragging() {
+        this.env.dashboardState.isDragging = false;
+    }
```

Comment:
```suggestion
    setDragging(value) {
        this.env.dashboardState.isDragging = value;
    }
```
What about something like this ?

We create the function to use 2 times for the disable but we also have 2 times for enable so why not do something like this instead ?